examples/vertical_data_loading_with_psi/DO1_PSI.ipynb

```diff
@@ -150,33 +219,36 @@
   {
    "cell_type": "code",
    "execution_count": 106,
-   "id": "quarterly-plain",
```

Comment:
interesting... I didn't intentionally delete this metadata. @daler3 what package are you using which is generating these?